*  datafusion/ERR/2022_007: Cloud Data Fusion Service Account exists
   - gcpdiag-datafusion1-aaaa/us-central1/my-instance                     [ OK ]

